metals-bench/src/main/scala/bench/MetalsBench.scala

```diff
@@ -154,4 +154,12 @@ class MetalsBench {
     )
   }
 
+  @Benchmark
+  @BenchmarkMode(Array(Mode.SingleShotTime))
+  def alltoplevelsScalaIndex(): Unit = {
```

Comment:
I checked the benchmarks on my machine also and the difference seems minimal, great work!